lib/smart_answer_flows/landlord-immigration-check.rb

```diff
@@ -6,6 +6,7 @@ def define
       status :published
```

Comment:
Could you please update the commit notes with a link to the Google Docs containing the flow?